qemu-guest-agent.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=QEMU Guest Agent
 IgnoreOnIsolate=True
+ConditionVirtualization=apple
```

Comment:
As per `man systemd.unit` looks like this is added in v244, and current used version in OCP is 252 which is good for this usecase. 

```
      ConditionVirtualization=
           Check whether the system is executed in a virtualized environment and optionally test whether it is a specific
           implementation. Takes either boolean value to check if being executed in any virtualized environment, or one of
           "vm" and "container" to test against a generic type of virtualization solution, or one of "qemu", "kvm",
           "amazon", "zvm", "vmware", "microsoft", "oracle", "powervm", "xen", "bochs", "uml", "bhyve", "qnx", "apple",
           "sre", "openvz", "lxc", "lxc-libvirt", "systemd-nspawn", "docker", "podman", "rkt", "wsl", "proot", "pouch",
           "acrn" to test against a specific implementation, or "private-users" to check whether we are running in a user
           namespace. See systemd-detect-virt(1) for a full list of known virtualization technologies and their
           identifiers. If multiple virtualization technologies are nested, only the innermost is considered. The test may
           be negated by prepending an exclamation mark.

           Added in version 244.
```